### ISSUE TITLE
z-index in CSS is too low

### ISSUE THREAD

**Issue by beruic** [dimsemenov/PhotoSwipe#1376]
Some CSS frameworks use relatively high z-indexes to have their menus behave correctly.
For example I currently work with a bootstrap theme/extension called Inspinia, where the sidebar menu has a z-index of a wooping 2001 :scream: 

But PhotoSwipe must be top most! :heart: 

Setting a higher z-index on the `pswp` class (line 16) fixes the problem, but I guess `$pswp__root-z-index` should be set in the . I guess if you set it to 15000 we will never see this issue again, so I am creating a pull request with this change.

**Comment by beruic** [comment]
I could really use this in a release, as we just moved to using yarn for third part frontend lib management, so my fixes were overwritten.

**Comment by dimsemenov** [comment]
Consider importing Sass files via your package manager from `src/css` directory, and not directly CSS. `$pswp__root-z-index` Sass variable is created for such cases.

I agree with you that the base z-index should be increased, but I don't want to do this until the major release, as this change might affect the appearance of some websites.

**Comment by beruic** [comment]
I agree with your thoughts. I have made our gulp project build photoswipe from source with overridden variables for now.
